README.md

```diff
@@ -1,8 +1,47 @@
+# Vandana Damodara . Jarvis Consulting
 
+I am a qualified and experienced software engineer with three years of experience in the industry. I had completed my  Bachelor of Technology degree in Electronics and Telecommunications. As a software developer, I believe that being able to work with/understanding lower levels of abstraction behind a program is a prerequisite to writing highly optimized code. I enjoy creative problem solving and getting exposure to multiple projects. I love working in a collaborative work environment. I am a Highly proficient, diligent, and excellent problem solver. As a Junior Data Engineer at Jarvis, I was able to participate in various challenging projects using an Agile/Scrum framework that allowed me to become proficient in technologies like Java, Git, SQL, Bash, Maven, and Docker. My experiences have taught me how to think outside the box when working, and I always strive to find the most optimized solution to the problem. I strive to stay current with relevant techniques and technologies
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** Hive, Spark, Hadoop, C++, Python, Hadoop/HDFS
+
+**Familiar:** MySQL, Scala, NoSQL, VBA, Unix
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_VandanaDamodara](https://github.com/jarviscanada/jarvis_data_eng_VandanaDamodara)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_VandanaDamodara/tree/master/linux_sql)]: Developed Cluster Monitoring agent which is used to monitor and collect data of various host machines connected through a network. The data that is collected includes hardware specifications, activity, and performances of machines in the cluster. Data is stored in PostgreSQL Database which is an RDBMS database provisioned with Docker Containers. Bash Scripts are used to initialize the process by creating psql container using Docker. Bash scripts are also used to monitor the host usage and host information data. This system uses crontab to collect real-time data every minute to generate statistics regarding resource utilization. The Monitoring agent sends the data to a database for storage and further analysis.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_VandanaDamodara/tree/master/core_java)]:
+      
+  - JDBC App:  Constructed the JDBC application to perform CRUD operations using DAO and DTO design patterns on a PostgreSQL retail store's database
+  - Grep App:  Programmed with Java 8 and leveraged its latest features including the stream API and lambda functions to develop an application similar to the Linux built-in grep feature. The program searches recursively through a root directory to map out all files and finds the lines that match the user-specified Java regular expression. Compilation was handled with the Maven tool, and the final product was packaged into a Docker container using Dockerfiles.
```

Comment:
Programmed with Java 8 and leveraged its latest features, including the stream API and lambda functions, to develop an application similar to the Linux built-in grep feature. The program searches recursively through a root directory to map out all files and finds the lines that match the user-specified Java regular expression. The compilation was handled with the Maven tool, and the final product was packaged into a Docker container using Dockerfiles.